.pre-commit-config.yaml

```diff
@@ -1,23 +1,27 @@
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Thanks for adding this check but the url is wrong :)

Reply:
Can you expand on how it's wrong? Does treeherder maintain it's own fork of these or?

`pre-commit` runs and is happy with the URL.

Reply:
Apologies, user error on my part - I must have made a type-o when I took a look at it.